sky/resources.py

```diff
@@ -308,7 +293,9 @@ def _try_validate_instance_type(self):
             self._cloud = valid_clouds[0]
 
     def _try_validate_accelerators(self) -> None:
-        """Try-validates accelerators against the instance type."""
+        """Try-validates accelerators against the instance type and
```

Comment:
nit: Make this single line `Validates accelerators against the instance type and region/zones.`

Reply:
Done.